examples/xo_android_client/app/src/main/java/io/bitwise/sawtooth_xo/MainActivity.kt

```diff
@@ -2,11 +2,16 @@ package io.bitwise.sawtooth_xo
 
 import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
+import io.bitwise.sawtooth_xo.state.XoState
+import java.util.*
```

Comment:
Explicit imports give a reader a better idea of what is being used in this import. It appears that it is UUID.